schedule/yast/raid/raid1_sle_aarch64_lvm.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name:           lvm+raid1@aarch64
+description:    >
+  Validation of partitioning for raid1 on lvm
+  Installation of RAID1 using expert partitioner.
+vars:
+  RAIDLEVEL: 1
+  LVM: 1
+test_data:
+  !include: test_data/yast/raid1_lvm/lvm_raid1.yaml
+schedule:
+  - installation/isosize
+  - installation/bootloader_uefi
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning_raid
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/hostname_inst
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - installation/grub_test
+  - installation/first_boot
+  - console/validate_lvm_with_raid1
```

Comment:
you didn't delete it, but if you still want to keep it for some reason, rename this one too.